### PR TITLE
feat: Updated Templates Documentation

### PR DESCRIPTION
**Pull request by JamesACS**
Updating Templates docs for new Devbox and Sandbox menu configurations. 

Included are some updated images. The Sandbox Templates docs are identical to the Devbox ones as bar the use of Sandbox instead of Devbox.

This duplication of information is included to improve documentation navigation for new users.

### CODE REVIEW

**Review by filipelima18 — APPROVED** [review]
I committed a few improvements, most notably one that removes some very outdated GIFs in both pages. I suggest getting a review from Neco too before merging.

**Review by necoline — APPROVED** [review]

**Pull request merged**
